client/src/shared/components/Breadcrumbs/index.tsx

```diff
@@ -0,0 +1,65 @@
+/* eslint-disable multiline-ternary */
+import { useRouter } from 'next/router';
+import React, { Fragment } from 'react';
+
+interface pathObject {
+  text: string;
+  url: string;
+}
+
+interface BreadcrumbsProps {
+  paths: pathObject[];
+}
+
+const Breadcrumbs: React.FunctionComponent<BreadcrumbsProps> = ({
+  paths
+}: BreadcrumbsProps) => {
+  const { asPath } = useRouter();
+  return (
+    <div>
+      <nav className="flex" aria-label="Breadcrumb">
+        <ol className="inline-flex items-center space-x-1 md:space-x-3">
+          {paths?.map((path, index) => (
+            <Fragment key={index}>
+              <li>
+                <div className="flex items-center">
+                  {index === 0 ? (
+                    ''
+                  ) : (
+                    <svg
+                      aria-hidden="true"
+                      className="w-6 h-6 text-gray-400"
+                      fill="currentColor"
+                      viewBox="0 0 20 20"
+                      xmlns="http://www.w3.org/2000/svg"
+                    >
+                      <path
+                        fillRule="evenodd"
+                        d="M7.293 14.707a1 1 0 010-1.414L10.586 10 7.293 6.707a1 1 0 011.414-1.414l4 4a1 1 0 010 1.414l-4 4a1 1 0 01-1.414 0z"
+                        clipRule="evenodd"
+                      ></path>
+                    </svg>
+                  )}
```

Comment:
- as much as possible. optimize the code by excluding empty condition
```suggestion
                  {index != 0 && (
                    <svg
                      aria-hidden="true"
                      className="w-6 h-6 text-gray-400"
                      fill="currentColor"
                      viewBox="0 0 20 20"
                      xmlns="http://www.w3.org/2000/svg"
                    >
                      <path
                        fillRule="evenodd"
                        d="M7.293 14.707a1 1 0 010-1.414L10.586 10 7.293 6.707a1 1 0 011.414-1.414l4 4a1 1 0 010 1.414l-4 4a1 1 0 01-1.414 0z"
                        clipRule="evenodd"
                      ></path>
                    </svg>
                  )}
```